src/scripts/settings/webSettings.js

```diff
@@ -55,30 +56,13 @@ async function getConfig() {
         return data;
     } catch (error) {
         console.warn('failed to fetch the web config file:', error);
-        return getDefaultConfig();
-    }
-}
-
-async function getDefaultConfig() {
-    try {
-        const response = await fetchLocal('config.template.json', {
-            cache: 'no-cache'
-        });
-
-        if (!response.ok) {
-            throw new Error('network response was not ok');
-        }
-
-        data = await response.json();
-        return data;
-    } catch (error) {
-        console.error('failed to fetch the default web config file:', error);
+        return DefaultConfig;
```

Comment:
It prints 404 error on each `getConfig`.
```suggestion
        data = DefaultConfig;
        return data;
```